api-gateway/src/test/java/com/petclinic/bffapigateway/presentationlayer/ApiGatewayControllerTest.java

```diff
@@ -172,6 +175,46 @@ void createOwner(){
         assertEquals(owner.getCity(),"Johnston");
         assertEquals(owner.getTelephone(),"51451545144");
     }
+
+    @Test
+    void createVets(){
```

Comment:
While the code below is commented this test doesn't assert what we want it to assert i.e. it is only asserting the attributes of the class.